### ISSUE TITLE
All GA fails if autotrack.js not loaded

### ISSUE THREAD

**Issue by tordans** [googleanalytics/autotrack#171]
Is there a way to configure this plugins in a way that don't break the whole GA implementation in case the `autotrack.js` cannot be found/loaded?

Our issue: We added the `autotrack.js` to one of our main files but not all – our mistake.
The result: GA did not track anything for those pages that did not have `autotrack.js` loaded. In addition the error is hard to spot since the GA Debug-Plugin shows debugging information but the message that shows why the `collect`-calls are postponed is not very clear - it just says

> Executing Google Analytics commands.
Registered new plugin: ga(provide, "ecommerce", Function)
Running command: ga("require", "maxScrollTracker")
Waiting on require of "maxScrollTracker" to be fulfilled.

There seems to be no timeout or anything to mitigate such an error?

**Comment by philipwalton** [comment]
This is [how analytics.js plugins work](https://developers.google.com/analytics/devguides/collection/analyticsjs/using-plugins#waiting_for_plugins_to_load), which means unfortunately there's no way to fix or address it in autotrack. If you like, you [file a bug](https://b.corp.google.com/issues?q=componentid:187400) against analytics.js.
